components/CatCard.tsx

```diff
@@ -127,56 +130,73 @@ export const CatCard = (props: CatCardProps) => {
           </View>
         </View>
-        {props.cat.notes && (
-          <Text style={styles.notes}>Observatii: {props.cat.notes}</Text>
+        {!!cat.notes && (
+          <Text style={styles.notes}>Observatii: {cat.notes}</Text>
         )}
-        {props.cat.isSterilized && (
+        {!!cat.isSterilized && (
           <View style={{ paddingTop: 8 }}>
             <View style={styles.informationLine}>
               <View style={styles.iconAndText}>
-                <Image source={leftArrow} style={styles.icon} />
+                <IconButton
+                  icon="calendar-arrow-left"
+                  size={20}
+                  color={theme.colors.placeholder}
+                  style={styles.icon}
+                />
                 <Text style={styles.baseText}>Dată internare:</Text>
               </View>
               <View style={{ width: '50%', justifyContent: 'flex-start' }}>
                 <Text style={styles.infoText}>
-                  {getDateText(props.cat.checkInDate)}
+                  {getDateText(cat.checkInDate)}
                 </Text>
               </View>
             </View>
             <View style={styles.informationLine}>
               <View style={styles.iconAndText}>
-                <Image source={rigthArrow} style={styles.icon} />
+                <IconButton
+                  icon="calendar-arrow-right"
+                  size={20}
+                  color={theme.colors.placeholder}
+                  style={styles.icon}
+                />
                 <Text style={styles.baseText}>Dată externare:</Text>
               </View>
               <View style={{ width: '50%', justifyContent: 'flex-start' }}>
                 <Text style={styles.infoText}>
-                  {getDateText(props.cat.checkOutDate)}
+                  {getDateText(cat.checkOutDate)}
                 </Text>
               </View>
             </View>
             <View style={styles.informationLine}>
               <View style={styles.iconAndText}>
-                <Image source={user} style={styles.icon} />
+                <IconButton
+                  icon="account"
+                  size={20}
+                  color={theme.colors.placeholder}
+                  style={styles.icon}
+                />
                 <Text style={styles.baseText}>Voluntar:</Text>
               </View>
               <View style={{ width: '50%', justifyContent: 'flex-start' }}>
-                <Text style={styles.infoText}>{props.cat.capturedBy.name}</Text>
+                <Text style={styles.infoText}>
+                  {cat.capturedBy?.name || ''}
+                </Text>
               </View>
             </View>
           </View>
         )}
-        {!isEmpty(props.cat.media) && (
+        {!isEmpty(cat.media) && (
           <FlatList
             style={{ marginTop: 20 }}
             horizontal
-            data={props.cat.media}
+            data={[] /*cat.media*/}
```

Comment:
[] is added with a purpose?

Reply:
Yes, for now `cat.media` is not an array, it's an object with some links keys inside and the FlatList won't accept it. Until we have a better data model, I replaced with an empty array.